src/main/common/cookieProperties.ts

```diff
@@ -0,0 +1,10 @@
+export default class CookieProperties {
+
+  static getCookieParameters () {
+    return {
```

Comment:
this doesn't look like secure has been set? or sameSite?

Reply:
we usually use the cookies library which sets cookies secure by default, (doesn't do sameSite though)
https://github.com/pillarjs/cookies

Reply:
have put in sameSite. cookieParser uses Cookie underneath which has a sameSite option:
https://github.com/jshttp/cookie